.github/workflows/ci-build.yml

```diff
@@ -19,47 +18,30 @@ jobs:
       - uses: actions/checkout@v4
       - uses: wyvox/action-setup-pnpm@v3
       # type checking requires build
-      - run: pnpm prepack
-        working-directory: addon
+      - run: pnpm build
       - run: pnpm lint
 
   test:
     timeout-minutes: 10
-    name: "Tests: ${{ matrix.EMBROIDER && 'embroider' || 'classic' }}"
```

Comment:
there is no longer any build behavior, so we aren't testing embroider and classic in a matrix -- we expect that embroider handles any behaviors used by this library